habitat/sims/habitat_simulator.py

```diff
@@ -433,12 +444,14 @@ def set_agent_state(
             of unit quaternion (versor) representing agent 3D orientation,
             (https://en.wikipedia.org/wiki/Versor)
             agent_id: int identification of agent from multiagent setup.
+            reset_sensors: bool for if sensor changes (e.g. tilt) should be
+                reset).
         """
         agent = self._sim.get_agent(agent_id)
         state = self.get_agent_state(agent_id)
         state.position = position
         state.rotation = rotation
-        agent.set_state(state)
+        agent.set_state(state, reset_sensors)
```

Comment:
Is there a use case for `reset_sensors = False`?

Reply:
The major use-case for it (preserving sensor positions through agent reconfigures) will be most depreciated after habitat-sim#6 lands as that expose the six DOF pose of sensors and allows that to be configured, but there may still be a use-case for it.